modules/Microsoft.VirtualMachineImages/imageTemplates/.test/common/deploy.test.bicep

```
@@ -11,7 +11,7 @@ param resourceGroupName string = 'ms.virtualmachineimages.imagetemplates-${servi
 param location string = deployment().location
 
 @description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
-param serviceShort string = 'vmicom'
+param serviceShort string = 'vmitcom'
```

```suggestion
param serviceShort string = 'vmiitcom'
```
I think it would even need to be this to be correct

---

honestly i only changed it in my repo due to an issue with the pre-existing staging resource group. i'll commit the change.